PLAY_SCREEN_IMPLEMENTATION.md

```
@@ -0,0 +1,269 @@
+# Play Screen Implementation Guide
+
+## Changes needed in `internal/ui/play.go`
+
+### 1. Update the Update() method to handle tick messages and help model size
+
+Add this at the beginning of the `Update()` method, right after the dimension checks:
+
+```go
+func (m PlayModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
+	// Check if we need to initialize models with dimensions we now have
```

markdownlint-cli2 (0.20.0)

[warning] Hard tabs
Column: 1

(MD010, no-hard-tabs)